File: CoronaBackend/src/main/java/business/model/CovidStat.java
package business.model;

import java.util.Date;

public class CovidStat
{
	private DataOrigin dataOrigin;
	private Date lastUpdated;
	private int statValue;

	public CovidStat(DataOrigin dataOrigin, Date lastUpdated, int statValue)
	{
		this.dataOrigin = dataOrigin;
		this.lastUpdated = lastUpdated;
		this.statValue = statValue;
	}

	public int getStatValue()
	{
		return statValue;
	}

	public void setStatValue(int statValue)
	{
		this.statValue = statValue;
	}

	public DataOrigin getDataOrigin()
	{
		return dataOrigin;
	}

	public void setDataOrigin(DataOrigin dataOrigin)
	{
		this.dataOrigin = dataOrigin;
	}

	public Date getLastUpdated()
	{
		return lastUpdated;
	}

	public void setLastUpdated(Date lastUpdated)
	{
		this.lastUpdated = lastUpdated;
	}
}

File: CoronaFrontend/app/src/Core/Model/CovidStat.ts
import {DataOrigin} from "./DataOrigin"

export class CovidStat
{
    public constructor(private _dataOrigin : DataOrigin, private _lastUpdated : Date, private _statValue : number)
    {

    }

    public get dataOrigin() : DataOrigin
    {
        return this._dataOrigin;
    }

    public set dataOrigin(dataorigin : DataOrigin)
    {
        this._dataOrigin = dataorigin;
    }

    public get lastUpdate() : Date
    {
        return this._lastUpdated;
    }

    public set lastUpdate(lu : Date)
    {
        this._lastUpdated = lu;
    }

    public get statValue() : number
    {
        return this._statValue;
    }

    public set statValue(counter : number)
    {
        this._statValue = counter;
    }

}